src/ssl.c

```diff
@@ -865,6 +865,31 @@ int wolfSSL_SetMinEccKey_Sz(WOLFSSL* ssl, short keySz)
     ssl->options.minEccKeySz = keySz / 8;
     return SSL_SUCCESS;
 }
+
+int wolfSSL_GetEccPrivateKey(WOLFSSL* ssl, struct ecc_key** key)
+{
+    if (ssl == NULL || key == NULL) {
+        return BAD_FUNC_ARG;
+    }
+
+    if (ssl->options.side == WOLFSSL_CLIENT_END) {
+        if (!ssl->peerEccKeyPresent) {
+            return NO_PRIVATE_KEY;
+        }
+
+        *key = (struct ecc_key*)ssl->peerEccKey;
```

Comment:
peerEccKey on client side case doesn't seem to match function name "PrivateKey".